clang/test/CodeGen/arm_acle.c

```diff
@@ -164,6 +164,19 @@ void test_pld() {
   __pld(0);
 }
 
+#if defined(__ARM_64BIT_STATE)
+
+// AArch64-LABEL: @test_rpld(
+// AArch64-NEXT:  entry:
+// AArch64-NEXT:    call void @llvm.aarch64.range.prefetch(ptr null, i32 1, i32 1, i32 15, i32 -2048, i32 65536, i32 2040)
+// AArch64-NEXT:    ret void
+//
+void test_rpld() {
+  __rpld(1, 1, 15, -2048, 65536, 2040, 0);
```

Comment:
nit, maybe change access kind to 0, so we can check against retention policy, because both can be 0 or 1.